stream-chat-android-ui-components/src/main/res/layout/stream_ui_message_list_view.xml

```diff
@@ -1,11 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<merge xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Some users would like to control the background of this view separately 🤔. We need to think about that.

Reply:
mmmm, that is exactly my use case. I need the `MessageListView` to have a null background. I am a bit unsure about why this PR does not allow users to do it?

Reply:
I misunderstand your PR. I thought you were merging the views with the parent ConstraintLayout in `fragment_chat.xml`. Yep, merging with the ConstraintLayout of `MessageListView` makes total sense and it is way better